### PR TITLE
Add option to skip when prolonging rental

### PR DESCRIPTION
**Pull request by AdamLazarus**
This fixes issue #414 by adding a 'Do not ask me again' option when prolonging a rental.

### CODE REVIEW

File: opacclient/opacapp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -258,6 +258,9 @@
     <string name="reservation_fee_warning">Please note that, depending on your library\'s rules,
         fees might apply for reservations. If in doubt, please check your library\'s scale of
         fees.</string>
+    <string name="prolong_fee_warning">Please note that, depending on your library\'s rules,
+        fees might apply for prolonging a reservation. If in doubt, please check your library\'s scale of
```
Comment (raphaelm) [inline opacclient/opacapp/src/main/res/values/strings.xml]: It's not "prolonging a reservation", its "prolonging an item" :)

File: opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/OpacApi.java [hunk 2]
```diff
@@ -86,6 +86,11 @@
     public static final int SUPPORT_FLAG_WARN_RESERVATION_FEES = 0x0000020;
 
     /**
+     * Asks the user responsibly about prolong fees
+     */
+    public static final int SUPPORT_FLAG_WARN_PROLONG_FEES = 0x0000030;
```
Comment (raphaelm) [inline opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/OpacApi.java]: This is used by a bitmask and therefore needs to be a value that only has one bit set, just choose ``0x0000040``.

**Comment by raphaelm** [comment]
Hi,

thank you very much for this PR! I added two minor remarks in the commit, otherwise it looks fine to me. I'll hope to actually try it out later today.

You somehow accidentally deleted our Android Studio configuration. No worries, that is more likely to be your fault than ours. I'll look into possible reasons and fixes later.

Cheers
Raphael

**Comment by raphaelm** [comment]
Sorry, I mistyped. I meant 'more likely to be *our* fault than *yours*'. Sorry!

**Comment by raphaelm** [comment]
I merged your pull request in c8c12fa. It will be part of the next version of the app. Thank you very much, your contribution is much appreciated!

**Comment by AdamLazarus** [comment]
Thanks very much! I'm pleased to have made a contribution.

**Pull request closed without merging**